package.json

```diff
@@ -7,29 +7,39 @@
     "test": "test"
   },
   "engines": {
-    "node": ">=0.12.0"
+    "node": ">=4.0.0"
   },
   "dependencies": {
+    "bluebird": "^3.2.1",
     "pluralize": "1.2.x",
     "qs": "5.2.x",
     "request": "2.9.x",
     "singleton": "1.0.x"
   },
   "devDependencies": {
-    "coffee-script": "1.10.x",
-    "docco": "0.7.x",
+    "coveralls": "^2.11.6",
```

Comment:
I prefer to install and run this in travis `after_script` hook. See https://github.com/lpinca/shopify-token/blob/b2ce543e1f84de9cfc0acc6d661ead4f11107409/.travis.yml#L11-L12.